test/parallel/test-fs-write-file-uint8array.js

```diff
@@ -17,13 +17,18 @@ const s = '南越国是前203年至前111年存在于岭南地区的一个国家
           '历经五代君主。南越国是岭南地区的第一个有记载的政权国家，采用封建制和郡县制并存的制度，' +
           '它的建立保证了秦末乱世岭南地区社会秩序的稳定，有效的改善了岭南地区落后的政治、##济现状。\n';
 
-const input = Uint8Array.from(Buffer.from(s, 'utf8'));
+const inputBuffer = Buffer.from(s.repeat(8), 'utf8');
```

Comment:
Could you add a comment explaining why `.repeat(8)` is necessary? Specifically, that `getArrayBufferViews` expects it.